index.js

```diff
@@ -67,6 +87,27 @@ module.exports = function (src, dest, opts) {
 				resolve(true);
 			});
 
+			read.on('data', function () {
```

Comment:
> Why are we listening to the read stream and not the write stream here? Just curious.

because there are no `data` events for writable streams. Any better alternatives?